pkg/storaged/block/format-dialog.jsx

```diff
@@ -380,30 +381,7 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
                                         })
                           ]
                       }),
-            SelectOne("at_boot", _("At boot"),
-                      {
-                          visible: is_filesystem,
-                          value: at_boot,
-                          explanation: mount_explanation[at_boot],
-                          choices: [
-                              {
-                                  value: "local",
-                                  title: _("Mount before services start"),
-                              },
-                              {
-                                  value: "nofail",
-                                  title: _("Mount without waiting, ignore failure"),
-                              },
-                              {
-                                  value: "netdev",
-                                  title: _("Mount after network becomes available, ignore failure"),
-                              },
-                              {
-                                  value: "never",
-                                  title: _("Do not mount"),
-                              },
-                          ]
-                      }),
+            at_boot_input(at_boot, is_filesystem),
             CheckBoxes("mount_options", _("Mount options"),
```

Comment:
Should this use `mount_options` as well?

Reply:
Good catch, I wasn't 100% sure if you'd like it so let's clean that up!